<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ana menü</string>
  </property>
  <widget class="QPushButton" name="btKisiEkle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Kişi Ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btYuzKaydet">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Yüz Kaydet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btYuzTani">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Yüz Tanı</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDurum">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>270</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Durum:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
